City-of-Helsinki/employment-services-ui
# Update packages

Update drupal-next and HDS packages.
Add sharp for prod images. More info: https://nextjs.org/docs/messages/sharp-missing-in-production

How to test:
```
yarn install
yarn start
```
This PR should be tested with this PR: https://github.com/City-of-Helsinki/drupal-employment-services/pull/92
